aten/src/ATen/functorch/BatchRulesReduceOps.cpp

```diff
@@ -405,6 +405,26 @@ static std::tuple<Tensor,optional<int64_t>> searchsorted_batch_rule(
   TORCH_INTERNAL_ASSERT(false);
 }
 
+Tensor bucketize_decomp_Tensor(
+    const Tensor& self,
+    const Tensor& boundaries,
+    bool out_int32,
+    bool right) {
+  // checking logical rank
+  TORCH_CHECK(boundaries.dim() == 1, "bucketize: boundaries tensor must be 1 dimension, but got dim(", boundaries.dim(), ")");
+  return at::searchsorted(boundaries, self, out_int32, right, nullopt, nullopt);
+}
```

Comment:
Could you add a manual test for this please? Specifically I want to make sure we are testing the case where `boundaries` is not a repeated tensor and our automated vmap tests always repeat the tensor.